packages/devextreme/js/__internal/ui/m_validation_engine.ts

```diff
@@ -562,10 +563,11 @@ const ValidationEngine = {
     this.addGroup();
   },
 
-  addGroup(group) {
+  addGroup(group, isRemovable) {
     let config = this.getGroupConfig(group);
     if (!config) {
       config = new GroupConfig(group);
+      config.isRemovable = isRemovable;
```

Comment:
[broken encapsulation]
We should not set object value out of the object, it's supposed to be done in the constructor.

Reply:
[9cf2b5d](https://github.com/DevExpress/DevExtreme/pull/27435/commits/9cf2b5dbbeeebb0ab7f8c03aa544c69a576b74ea)